clients/python-client/python_client/utils/kuberay_cluster_builder.py

```diff
@@ -121,17 +120,14 @@ def build_worker(
         group_name: str,
         ray_image: str = "rayproject/ray:2.4.0",
         ray_command: Any = ["/bin/bash", "-lc"],
-        init_image: str = "busybox:1.28",
```

Comment:
@jasoonn @akanso is this a user-facing function? Thanks!

Reply:
Yes. It gives users the ability to customize their workers' configuration.

Reply:
The build_worker() passes init_image as a parameter to call populate_worker_group() to generate worker group configuration. However, populate_worker_group() does not actually use that parameter. Hence, it makes sense to me to remove init_image here.

Reply:
If `build_worker` is a user-facing function, this change seems to break the backward compatibility. Is it OK?

Reply:
It's not okay. I forgot to take backward compatibility into consideration.

Reply:
Let's keep init_image unchanged on this PR and make a separate PR to discuss this issue.